server/routes/github_oauth.js

```diff
@@ -351,6 +351,23 @@ async function addNewProject(auth_token, position, experience_lvl, new_project)
     });
 }
 
+async function deleteProject(id) {
+  let SQL = `DELETE FROM projects WHERE id=${id};`;
+  await client.query(SQL)
+    .then(async (res) => {
+      return res;
+    })
+    .catch(err => {
+      throw err;
+    });
+};
+
+router.post("/projects/:id/delete", async (req,res) => {
+  const response = await deleteProject(req.params.id);
```

Comment:
Let's make sure to add check for authentication before deleting the project 😛. Please add following after L365:
 ```
 const auth_token = getToken(req.headers.authorization);
  if (auth_token === null) {
    return res.sendStatus(401);
  }
```

Reply:
Good call! I added that in 👍